BGL/include/CGAL/boost/graph/convert_nef_polyhedron_to_polygon_mesh.h does not compile

@sloriot 
```
/cgal/Installation/../BGL/include/CGAL/boost/graph/convert_nef_polyhedron_to_polygon_mesh.h:
In member function ‘void CGAL::nef_to_pm::Shell_polygons_visitor<Nef_polyhedron>::visit(typename Nef_polyhedron::Halffacet_const_handle)’:
/cgal/Installation/../BGL/include/CGAL/boost/graph/convert_nef_polyhedron_to_polygon_mesh.h:101:7:  error: expected ‘;’ before ‘polygons’
       polygons.back().push_back(vertex_indices[hc_start->source()->center_vertex()]);
       ^~~~~~~~
```
The previous line is
```C++
    CGAL_For_all(hc_start,hc_end)
```
Probably a missing `#include` for `CGAL_For_all`.

--- a/BGL/include/CGAL/boost/graph/convert_nef_polyhedron_to_polygon_mesh.h
+++ b/BGL/include/CGAL/boost/graph/convert_nef_polyhedron_to_polygon_mesh.h
@@ -24,6 +24,7 @@
 #include <CGAL/boost/graph/helpers.h>
 #include <CGAL/algorithm.h>
 #include <CGAL/Polygon_mesh_processing/polygon_soup_to_polygon_mesh.h>
+#include <CGAL/circulator.h>
 #include <boost/unordered_map.hpp>
 
 namespace CGAL{

